frontend/src/views/CensusDirectory/index.tsx

```diff
@@ -17,92 +40,196 @@ function CensusDirectory() {
   `;
 
   const Header = styled.h1`
+    ${fontHeaderXxl}
     margin-bottom: 8px;
     font-weight: 700;
-    ${fontHeaderXxl}
   `;
 
   const Paragraph = styled.p`
-    font-weight: 400;
     ${fontBodyS}
+    font-weight: 400;
+    margin-bottom: 0;
+  `;
+
+  const DirectoryDescription = styled(Paragraph)`
+    margin-bottom: 80px;
+  `;
+
+  const TierContainer = styled.div`
+    margin-bottom: 120px;
   `;
 
   const TierTitle = styled.h3`
+    ${fontHeaderXl}
     margin-bottom: 8px;
     font-weight: 600;
-    ${fontHeaderXl}
+  `;
+
+  const TierDescription = styled.p`
+    ${fontBodyS}
+    color: #767676;
```

Comment:
`${defaultTheme.palette.text.secondary};`

We should probably add `text` to the `getColors` function instead of having to access the default theme file directly.